generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -361,6 +382,35 @@ public void initTest() {<% if (databaseType == 'mongodb' || databaseType == 'cas
 
         // Validate the database is empty
         List<<%= entityClass %>> <%= entityInstancePlural %> = <%= entityInstance %>Repository.findAll();
-        assertThat(<%= entityInstancePlural %>).hasSize(databaseSizeBeforeDelete - 1);
-    }
+        assertThat(<%= entityInstancePlural %>).hasSize(databaseSizeBeforeDelete - 1);<% if (searchEngine == 'elasticsearch') { %>
+
+        // Validate tElasticSearch is empty
+        boolean <%= entityInstance %>ExistsInEs = <%= entityInstance %>SearchRepository.exists(<%= entityInstance %>.getId());
```

Comment:
When entityInstance is same that entityInstancePlural, there is a problem here (with entity people for example). 

`<%= entityInstance %>.getId()` does not exist because entityInstance was redefined in a `List<<%= entityClass %>>`